### PR TITLE
STM32: Board conversion to DT clock configuration. STM32L5

### PR DESCRIPTION
**Pull request by ABOSTM**
Following #34609, Board conversion to DT clock configuration. STM32L5 boards.
Compilation OK. Not tested (I don't have the boards).

* stm32l562e_dk

### CODE REVIEW

**Review by galak — APPROVED** [review]

**Review by FRASTM — APPROVED** [review]

File: boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi [hunk 1]
```diff
@@ -51,6 +51,33 @@
 	};
 };
 
+&clk_msi{
+	status = "okay";
+	msi-range = <6>;
+};
```
Comment (erwango) [inline boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi:57]: Miss:
```
       msi-pll-mode;
```

**Review by erwango — CHANGES_REQUESTED** [review]

**Review by erwango — APPROVED** [review]

File: boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi (hunk 1 above)
Comment (ABOSTM) [reply boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi:58]: Nice catch, thanks. Done.

**Comment by ABOSTM** [comment]
Wrong push. Reopen PR.

**Review by MaureenHelm — APPROVED** [review]

File: boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi [hunk 2]
```diff
@@ -51,6 +51,34 @@
 	};
 };
 
+&clk_msi{
```
Comment (tagunil) [inline boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi:54]: These whitespaces are not very consistent, so I think we'll need a cleaning pass after this mass conversion. @erwango, do we have something like checkpatch for dts files?

**Comment by ABOSTM** [comment]
Rebase to force buildkite execution.

File: boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi (hunk 2 above)
Comment (erwango) [reply boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi:54]: We do (dtc), but unfortunately, it doesn't catch this.

**Pull request merged**